src/components/NetworkListTable.tsx

```diff
@@ -3,35 +3,38 @@ import { useQuery } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
 import { MainTable } from "@canonical/react-components";
 import Loader from "components/Loader";
-import { LxdInstance } from "types/instance";
 import { fetchNetworks } from "api/networks";
 import { isNicDevice } from "util/devices";
 import ResourceLink from "components/ResourceLink";
+import { useParams } from "react-router-dom";
+import { LxdDevices } from "types/device";
 
 interface Props {
-  instance: LxdInstance;
   onFailure: (title: string, e: unknown) => void;
+  networkDevices: LxdDevices;
```

Comment:
```suggestion
  devices: LxdDevices;
```
There are all devices, not just network devices. Better reflect that in the name not to confuse later on. Also it is shorter and thus simpler.